src/orquestra/sdk/secrets/_api.py

```diff
@@ -11,16 +11,25 @@
 from . import _auth, _exceptions, _models
 
 
+def _translate_to_zri(workspace_id: str, secret_name: str) -> str:
+    """
+    Create ZRI from workspace_id and secret_name
+    """
+    return f"zri:v1:unused:0:{workspace_id}:secret:{secret_name}"
```

Comment:
`unused` shouldn't be there, I think. It should just be empty. See `cluster` section in this doc: https://zapatacomputing.atlassian.net/wiki/spaces/Platform/pages/512787664/2022-09-26+Zapata+Resource+Identifiers+ZRIs
```suggestion
    return f"zri:v1::0:{workspace_id}:secret:{secret_name}"
```